src/js/media-live-button.js

```diff
@@ -8,7 +8,7 @@ const { MEDIA_TIME_IS_LIVE, MEDIA_PAUSED } = MediaUIAttributes;
 const { MEDIA_SEEK_TO_LIVE_REQUEST, MEDIA_PLAY_REQUEST } = MediaUIEvents;
 
 const indicatorSVG =
-  '<svg viewBox="0 0 6 12"><circle cx="3" cy="6" r="2"></circle></svg>';
+  '<svg viewBox="0 0 4 4" style="width:6px; height:6px;"><circle cx="2" cy="2" r="2"></circle></svg>';
```

Comment:
can this be done without the style attribute? 
might as well change everything in SVG